### ISSUE TITLE
Final Task do(code)

### ISSUE THREAD

**Issue by danaderp** [WM-SEMERU/ds4se#117]
Here is the final list of activities for do(code):

- [ ] Please prepare the granularity datasets to be compatible with our data frames. Follow the same structure of data frames as LibEST. 
- [ ] Although we do not need to run word2vec or doc2vec on the granularity datasets, it is important to generate a preprocessed version. There is a preprocessing notebook that you can employ for this task. 
- [ ] Please generate a causal model where the treatment is the granularity variability. You can analyze performance based on the outcomes of TC  Tracer (ICSE2020)
- [ ] Please explain the step-by-step of how to model and evaluate a causal model in a nbdev notebook.

**Comment by scheurich-sarah** [comment]
By "Follow the same structure of data frames as LibEST" do you mean include the same fields?

**Comment by danaderp** [comment]
@scheurich-sarah Yes. Same fields and notebook. You need to create your own script to extract the info from the new sources though.

**Comment by scheurich-sarah** [comment]
@danaderp I couldn't figure out the pre-processing bit, so apologies, but the first 3 check marks aren't done. I read predicted and ground truth links files they provided into dfs and aggregated them across all the projects, but that's all.
